pkg/testgrid/testgrid.go

```diff
@@ -0,0 +1,106 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package testgrid
+
+import (
+	"io/ioutil"
+	"os"
+
+	"github.com/GoogleCloudPlatform/testgrid/config"
+	pb "github.com/GoogleCloudPlatform/testgrid/pb/config"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+
+	"k8s.io/release/pkg/http"
+)
+
+const testgridConfigURL = "https://storage.googleapis.com/k8s-testgrid/config"
```

Comment:
if that changes we need to rebuild, maybe have a flag to pass this and have the default value as this one.

this is just a comment, you can ignore :)